Web/starter_project_iconics/styles/globals.css

```diff
@@ -1,7 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;600;700;800&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;0,800;1,600&display=swap');
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
+  html {
+    font-family: 'Poppins', sans-serif;
+  }
+}
+
 :root {
```

Comment:
I would suggest removing css configurations that you're not using. I belive these are generated by default. If you're going to use them, it's okay you don't have to remove it but if not I don't see the need to have it.